src/EncompassRest/Loans/LoanEntity.cs

```diff
@@ -12,7 +12,6 @@ public enum LoanEntity
         ATRQMBorrower = 8,
         ATRQMCommon = 9,
         Borrower = 10,
-        CoBorrower = 11,
```

Comment:
why is this dropping off?

Reply:
Because CoBorrower is not actually a loan entity type. When retrieving the loan schema for CoBorrower you get the Borrower entity's schema.

Reply:
it is listed as a separate entity in the schema

Reply:
See https://forum.elliemae.com/index.php?/topic/6271-loan-entity-schema-issues/

Reply:
so, ultimately....are we making something that matches the em system, or making something that just works?

Reply:
Ellie Mae's system is wrong so we shouldn't follow it strictly. Hopefully they'll address those issues as I described in the forum.

Reply:
ok.  we need to docume t the deviations in a git file.

Reply:
Do you think the Loan_GetSupportedEntities unit test is enough for documentation purposes. See [here](https://github.com/EncompassRest/EncompassRest/pull/67/files#diff-568fb4ee1eb8fb0f37ab6a77a5d32d07R24).

Reply:
something simple like:

# EncompassRest .NET Library Deviations

## Schema entries that deviate from documentation
CoBorrower - see Borrower
LOCompensation - see ElliLOCompensation
ElliUCDFields -see ElliUCDDetail
DocumentOrderLog
NonVols - see NonVol

Reply:
I think that's a bit overboard as you can still specify those values in whatever api methods accepts a LoanEntity enum value.

Reply:
think about this from the not us perspective.  a new user comes to use this, looks for the coborrower entity because it is listed in ellie mae's documentation, and cannot find it in ours.  they will report it as a bug, but if we have a simple list of places we corrected for ellie maes errors, we wont have to remember.  simmilar to the issue we are having with the webhook.  we have to deviate because of x,y,z...

Reply:
I view this enum as a convenience enum which is likely to become out of sync whenever Encompass adds new entities and we haven't updated and released a new version of EncompassRest or the user hasn't upgraded yet. I think if it comes up we can address it then but if not, why bother. You're free to add this document though in a separate PR if you'd like.